model/messaging/spans.yaml

```
@@ -311,3 +311,23 @@ groups:
 
           If none of the above operation names apply, the attribute SHOULD be set
           to the name of the client method in snake_case.
+
+  - id: messaging.aws.sqs
+    type: attribute_group
+    stability: development
+    extends: attributes.messaging.trace.minimal
+    brief: "Attributes that exist for SQS request types."
+    attributes:
+      - ref: aws.sqs.queue.url
+        requirement_level: recommended
+      - ref: messaging.destination.name
+        requirement_level: recommended
```

if you feel it's goo, please add
nit

```suggestion
        requirement_level: recommended
     - ref: aws.request_id
        requirement_level: recommended 
```

---

Added!